exec/bytecode/type.go

```diff
@@ -0,0 +1,40 @@
+package bytecode
+
+import (
+	"reflect"
+
+	"github.com/goplus/gop/exec.spec"
+)
+
+type Type struct {
```

Comment:
In exec/bytecode, don't need introduce a custom Type.

Reply:
We may have DefineType(name string, typ reflect.Type) interface, It just because for Go code generator of exec/golang.